packages/css/src/components/accordion/accordion.scss

```
@@ -34,20 +40,21 @@
   }
 
   &:hover {
-    box-shadow: var(--ams-accordion-button-hover-box-shadow);
+    color: var(--ams-accordion-button-hover-color);
   }
+}
 
-  svg {
-    transform: rotate(0deg);
-    transition: transform 0.3s ease;
+.ams-accordion__icon svg {
+  transform: rotate(0deg);
+  transition: transform 0.3s ease;
 
-    @media (prefers-reduced-motion) {
-      transition: none;
-    }
+  [aria-expanded="true"] & {
+    transform: rotate(180deg);
   }
 
-  @include text-rendering;
-  @include reset-button;
+  @media (prefers-reduced-motion) {
+    transition: none;
+  }
```

A simpler pattern could be to set a transition only once – if people do not prefer reduced motion.

```suggestion
  @media not (prefers-reduced-motion) {
    transition: transform 0.3s ease;
  }
```